src/all/novelfull/src/eu/kanade/tachiyomi/extension/all/novelfull/NovelFullFilters.kt

```diff
@@ -0,0 +1,196 @@
+            SelectFilterOption("热血玄幻古风", "130"),
+            SelectFilterOption("彩虹都市", "121"),
+            SelectFilterOption("热血冒险", "131"),
+            SelectFilterOption("治愈", "108"),
+            SelectFilterOption("搞笑", "101"),
+            SelectFilterOption("古风", "115"),
+            SelectFilterOption("校园", "106"),
+            SelectFilterOption("猎奇", "114"),
+            SelectFilterOption("日常", "105"),
+            SelectFilterOption("同人", "116"),
+            SelectFilterOption("恋爱都市", "123"),
+            SelectFilterOption("纯爱", "535"),
+            SelectFilterOption("恋爱后宫搞笑", "128"),
+            SelectFilterOption("其它", "542"),
+            SelectFilterOption("生活", "135"),
+            SelectFilterOption("修真", "513"),
+            SelectFilterOption("恐怖", "111"),
+            SelectFilterOption("百合", "518"),
+            SelectFilterOption("奇幻冒险", "528"),
+            SelectFilterOption("耽美", "533"),
+            SelectFilterOption("恋爱古风", "126"),
+            SelectFilterOption("灵异", "540"),
+            SelectFilterOption("总裁", "545"),
+            SelectFilterOption("恋爱", "104"),
+            SelectFilterOption("后宫", "498"),
+            SelectFilterOption("玄幻", "109"),
+            SelectFilterOption("霸总", "516"),
+            SelectFilterOption("剧情", "526"),
+            SelectFilterOption("恋爱校园都市", "119"),
+            SelectFilterOption("少年", "531"),
+            SelectFilterOption("恋爱搞笑治愈", "124"),
+            SelectFilterOption("战争", "538"),
+            SelectFilterOption("校园日常", "129"),
+            SelectFilterOption("大女主", "543"),
+            SelectFilterOption("热血", "102"),
+            SelectFilterOption("架空", "496"),
+            SelectFilterOption("日漫", "107"),
+            SelectFilterOption("动作", "514"),
+            SelectFilterOption("悬疑", "112"),
+            SelectFilterOption("四格", "519"),
+            SelectFilterOption("热血悬疑都市", "117"),
+            SelectFilterOption("系统", "529"),
+            SelectFilterOption("校园搞笑", "122"),
+            SelectFilterOption("魔法", "534"),
+            SelectFilterOption("推理", "541"),
+            SelectFilterOption("热恋", "134"),
+            SelectFilterOption("真人", "546"),
+            SelectFilterOption("科幻", "499"),
+            SelectFilterOption("奇幻", "110"),
+            SelectFilterOption("欢乐向", "517"),
+            SelectFilterOption("励志", "527"),
+            SelectFilterOption("恋爱奇幻", "120"),
+            SelectFilterOption("少女", "532"),
+            SelectFilterOption("热血悬疑", "125"),
+            SelectFilterOption("竞技", "539"),
+            SelectFilterOption("格斗", "544"),
+            SelectFilterOption("冒险", "103"),
+            SelectFilterOption("穿越", "497"),
+            SelectFilterOption("[empty]", "515"),
+            SelectFilterOption("都市", "113"),
+            SelectFilterOption("武侠", "525"),
+            SelectFilterOption("玄幻都市", "118"),
+            SelectFilterOption("爱情", "530"),
+        ),
+    )
+    // TODO: Automate this
```

Comment:
I need help here

Reply:
you could maybe have a list of tripple and build the list based on language/index.

Reply:
Not sure for multiple languages, but you can use the selector: `ul.nav > li:has(> a:containsOwn(Genre)) ul > li`
with that you can get id and name kinda like this:
```kt
allGenres["en"] = document.select("ul.nav > li:has(> a:containsOwn(Genre)) ul > li > a").mapNotNull { element ->
    val url = element.absUrl("href").toHttpUrlOrNull()
    val id = url?.queryParameter("id")
    id?.let { SelectFilterOption(element.ownText(), it) }
}
```
(code can be considered pseudo code)